README.project.md

```diff
@@ -12,12 +12,16 @@ git clone {{git_url}}
 
 ## Install Dependencies
 
+Ensure that the dependencies are set-up:
+
 1. Install the latest version of Hem from https://github.com/inviqa/hem
 
 2. Install the latest version of Vagrant from [http://www.vagrantup.com/downloads.html](http://www.vagrantup.com/downloads.html)
 
 3. Ensure you have AWS keys correctly configured either using `hem config` or by setting the AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY environment variables.
 
+If you have performed any of these steps for other seeds/projects you do not need to perform them again here.
```

Comment:
technically this might not be true, Hem and Vagrant may need updating.

Vagrant will say so based on the version in the Vagrantfile
Hem won't atm, but from the next release Hemfiles can explicitly say their minimum version